### PR TITLE
cody docs: make setup process for admins clearer

### PR DESCRIPTION
**Pull request by mrnugget**
This makes some tiny changes to the docs for site-admins:

1. Use bullet points to make clear that there are 3 steps
2. Remove paragraph to say that embeddings can be configured, since they're enabled by default if `cody.enabled: true`
3. Make instructions consistent with others ("Go to", link style, ...)
4. Link to "Configuring embeddings" page

## Test plan

- Manual testing with `sg run docsite`

### CODE REVIEW

**Comment by ErikaRS** [comment]
Looks good. Thanks for adding clarity here!

**Comment by ErikaRS** [comment]
Urgh. Sorry. I didn't mean to close this. Wrong button.

**Review by ErikaRS — APPROVED** [review]

File: doc/cody/explanations/enabling_cody_enterprise.md [hunk 1]
```diff
@@ -21,20 +21,22 @@ There are two steps required to enable Cody on your enterprise instance:
...
+    ```json
+    {
+      // [...]
+      "cody.enabled": true
+    }
+    ```
+3. Setup a policy to automatically create embeddings for repositories: ["Configuring embeddings"](code_graph_context.md#configuring-embeddings)
 
-Cody can also be configured to use embeddings for code graph context to significantly improve the quality of its responses. This involves sending your entire codebase to a third-party service to generate a low-dimensional semantic representation, that is used for improved context fetching. See the [codebase-aware answers](#enabling-codebase-aware-answers) section for more.
+Cody is now fully setup on your instance!
```
Comment (bobheadxi) [inline doc/cody/explanations/enabling_cody_enterprise.md:39]:
```suggestion
Cody is now fully set up on your instance!
```

File: doc/cody/explanations/enabling_cody_enterprise.md [hunk 2]
```diff
@@ -21,20 +21,22 @@ There are two steps required to enable Cody on your enterprise instance:
...
-  "cody.enabled": true,
-}
-```
+    ```json
+    {
+      // [...]
+      "cody.enabled": true
+    }
+    ```
+3. Setup a policy to automatically create embeddings for repositories: ["Configuring embeddings"](code_graph_context.md#configuring-embeddings)
```
Comment (bobheadxi) [inline doc/cody/explanations/enabling_cody_enterprise.md:37]:
```suggestion
3. Set up a policy to automatically create embeddings for repositories: ["Configuring embeddings"](code_graph_context.md#configuring-embeddings)
```

**Review by bobheadxi — APPROVED** [review]
LGTM! Only minor nit ([`set up` is the verb](https://writingexplained.org/setup-vs-set-up-difference#:~:text=Both%20words%20have%20different%20meanings,in%20order%20or%20arranging%20them) 😛 )

**Pull request merged**